### DISCUSSION TITLE
Improve farming

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#698] [Ideas]
I would suggest to have farms capable of handling more fields based on the number of employees assigned to it like the farms in Emperor rise of the middle kingdom. In my last city (Carthago) I observed the natives and they have tiny one tile fields so what if we have bigger and bigger farms by adding fields to them instead of the normal fixed 8 fields. Many occasions I see wasted farm land where there is not enough room for a farm but there are 3x2 spaces for 6 tiny fields.

**Comment by crudelios** [comment]
This has been planned for a while actually, but won't be implemented for a while.
